pages/index.js

```diff
@@ -4,31 +4,48 @@ import TipOfTheDay from "@/components/TipOfTheDay";
 import styled from "styled-components";
 import { useState } from "react";
 import useSWR from "swr";
+import Image from "next/image";
+import ClickButton from "@/assets/ClickButton.png";
+import ReloadButton from "@/assets/ReloadButton.png";
+import React from "react";
 
 const Button = styled.button`
-  background-color: black;
-  color: #fff;
+  background: transparent;
+
   border: none;
   border-radius: 5px;
   padding: 10px 20px;
   cursor: pointer;
 `;
 const TipOfTheDayWrapper = styled.div`
-  text-align: center;
+  position: fixed;
+  z-index: 1;
+  bottom: 100px;
   display: flex;
   flex-direction: column;
-  align-items: center;
+`;
+
+const Backdrop = styled.div`
+  position: fixed;
+  top: 0;
+  width: 100vw;
```

Comment:
As far as im concerned, our coach dissuade us from using the vh vw at all, cause of possible layout issues. But maybe this is a use case for it, because its all at 100.

I just pondering if there is a solution using the object-fit: cover/fill property. That would only be one line, but im unsure about that.